### PR TITLE
Add Cursed Castilla EX

### PR DESCRIPTION
**Pull request by AuraOfTheDawn**
Adds Cursed Castilla EX.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/cursed-castilla-ex/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "Cursed Castilla EX"
+description = "Maldita Castilla (Cursed Castilla EX) is an arcade action video game developed by Locomalito (Assisted by Abylight on 3DS) and released in December 2012. (3DS version released in July 2017) The game is primarily based on myths from Spain and other parts of Europe. Maldita Castilla was developed as a tribute to the video games of the late 1980s, such as Ghosts'n Goblins, Tiger Road, Black Tiger, and Shinobi. The game was inspired by Amadis of Gaul, a sixteenth-century Spanish chivalric romance."
```
Comment (Schplee) [inline games/cursed-castilla-ex/game.dat]: Remove the release dates from the game description, as it's redundant.

**Comment by Leo626** [comment]
Can you report the black screen issue [here](https://github.com/citra-emu/citra/issues) (make sure you provide the necessary info so that it doesn't get closed) and add it to the game.dat (see [Contributing ](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md) guide on how to add it)?

File: games/cursed-castilla-ex/game.dat [hunk 2]
```diff
@@ -0,0 +1,21 @@
+title = "Cursed Castilla EX"
+description = "Maldita Castilla (Cursed Castilla EX) is an arcade action video game developed by Locomalito. (Assisted by Abylight on 3DS) The game is primarily based on myths from Spain and other parts of Europe. Maldita Castilla was developed as a tribute to the video games of the late 1980s, such as Ghosts'n Goblins, Tiger Road, Black Tiger, and Shinobi. The game was inspired by Amadis of Gaul, a sixteenth-century Spanish chivalric romance."
```
Comment (Leo626) [inline games/cursed-castilla-ex/game.dat]: Small nit with the first sentence. Prefer it to start like this:
```
Cursed Castilla EX (originally released as Maldita Castilla in 2012) is an arcade action video game developed by Locomalito with assistance by Abylight on 3DS.
```

**Comment by Schplee** [comment]
LGTM

**Pull request merged**